src/app/components/admin/dashboard/dashboard.component.ts

```diff
@@ -40,10 +41,8 @@ export const adminMenuItemActions = [
 
 @Component({
   selector: "baw-dashboard",
-  template: `
-    <h1>Admin Dashboard</h1>
-    <p>Welcome to the super-secret-site-settings!</p>
```

Comment:
why do you remove my fun?

Reply:
![image](https://github.com/user-attachments/assets/1628d149-dd25-437e-9c9f-97a4e2327004)